engine/src/main/java/org/hibernate/search/backend/spi/BackendQueueProcessor.java

```diff
@@ -60,17 +59,6 @@
 	void applyStreamWork(LuceneWork singleOperation, IndexingMonitor monitor);
 
 	/**
-	 * @return a Lock instance which will block index modifications when acquired
-	 */
-	Lock getExclusiveWriteLock();
```

Comment:
Not needed anymore, as `DirectoryBasedIndexManager` only needs this internally and invokes that functionality on the `WorkspaceHolder`.